chroma_help/help.py

```diff
@@ -196,4 +196,8 @@
     "revoke_ticket": "Revoking Ticket",
     "forget_ticket": "Forgetting Ticket",
     "create_task": "Creating Task",
+    "mount_snapshot": "Mounting Snapshot",
```

Comment:
Not really a big deal but the long descriptions are shorter than the standard descriptions. You could store the values that you have under `description` in this help.py file and then call them:
```
 @classmethod
    def long_description(cls, stateful_object):
        return help_text["unmount_snapshot"] % stateful_object.fqdn

    def description(self):
        return self.long_description(self)
```